quinn/src/tests.rs

```diff
@@ -384,6 +384,71 @@ async fn zero_rtt() {
     endpoint.wait_idle().await;
 }
 
+#[tokio::test]
+async fn zero_rtt_resumption() {
+    let _guard = subscribe();
+    let endpoint = endpoint();
+
+    let endpoint2 = endpoint.clone();
+    tokio::spawn(async move {
```

Comment:
Do we need to join this at the end of the test to ensure any panics are surfaced?